Source/Core/AudioCommon/AlsaSoundStream.cpp

```diff
@@ -20,7 +20,7 @@ AlsaSound::~AlsaSound()
   m_thread_status.store(ALSAThreadStatus::STOPPING);
 
   // Immediately lock and unlock mutex to prevent cv race.
-  std::unique_lock<std::mutex>{cv_m};
+  std::unique_lock<std::mutex>{cv_m}.release();
```

Comment:
Err, is that right? https://en.cppreference.com/w/cpp/thread/unique_lock/release doesn't unlock the mutex.

Reply:
My bad, I think you're correct, I thought `release()` also unlock the mutex. So I guess `unlock()` should be call instead.

Reply:
Thanks for catching that!